src/main/python/run_trecqa.py

```diff
@@ -150,6 +161,7 @@ def get_precision(actual, predicted, k):
   parser.add_argument("-k", help="top-k passages", default=10)
   parser.add_argument("-model", help="[idf|sm]", default='idf')
   parser.add_argument('-index', help="path of the index", required=True)
+  parser.add_argument('--qa-model-file', help="the path to the model file")
```

Comment:
AFAIK the https://docs.python.org/3/library/argparse.html convention is:
- no dashes are for required positional arguments. (script.py arg1 arg2)
- double dashes "--" are for switches that make the program change its operation. (script.py arg1 arg2 --run-it-this-way). Switches could have parameters  as well. (--num-iterations  4 or --use-this-file filename)
- single dash "-" is usually prescribed as a "shortcut" for switches. (-h for --help)

However I see that you have used the required=True flag. Note that the --qa-model-file is a switch, i.e. the program can run without it. 
Both approaches to arguments can work IMO. However, note that there now exist different conventions in Anserini and Castor. Do we want to keep them same? or, Do we want to follow respective conventions?